specification/repository.md

```diff
@@ -144,3 +144,9 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 - MUST document how to configure log correlation
 - MUST define minimum supported version of each auto-instrumentation framework
   - SHOULD define maximum supported version of each auto-instrumentation framework
+
+## Real User Monitoring Libraries
+
+- MUST document all supported configuration parameters and how to set them
+- MUST document how to configure manual instrumentation
+- MUST document supported instrumentations and telemetry produced by each of them
```

Comment:
I wonder how important this part is, since the low-level telemetry may or may not be particularly accessible in the RUM UI. (the telemetry part, not the supported instrumentation)

Reply:
Hmm, do you think it makes sense not to describe span attributes in details? Would just general instrumentation info (when it is called, what spans/names it produces) be enough?

Reply:
Is that something that even needs to be in each repository? I think I'd rather push all of that documentation to some kind of generalized RUM documentation site. At this point, I think users probably won't care very much about the details of the telemetry, compared to what they end up seeing in the RUM UI, etc.

Reply:
That's a good point. Let's stick to a brief instrumentation documentation in the repo